docs/per-env-values.md

```
@@ -0,0 +1,217 @@
+
+```bash
+braze-sync export --env=dev
+braze-sync export --env=staging
+```
+
+After that, `diff` / `apply` work as before — placeholders are resolved
+against `values/<env>.yaml` before any HTTP write.
+
+## Placeholder syntax
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)